Assets/Source/Player/InputController.cs

```diff
@@ -56,5 +56,28 @@ void Update ()
 
 		if(destination != originalPos)
 			Camera.main.transform.position = Vector3.MoveTowards(originalPos, destination, Time.deltaTime*Options.ScrollSpeed);
+
+		// Queen controls for some simple testing.
+		Transform queen = transform.root.Find("/Basic Unit 2");
+
+		if(queen) {
+			Vector3 displacement = new Vector3();
+
+			if(Input.GetKey(KeyCode.I))
```

Comment:
I realize this was for quick testing purposes but I like sticking to the convention Hazen came up with.
```
displacement.z += Input.GetAxis("MoveUp");
displacement.x += Input.GetAxis("MoveRight")
```